aten/src/ATen/native/sparse/cuda/cuSPARSELtOps.cpp

```diff
@@ -103,14 +91,16 @@ at::Tensor _cslt_compress(const Tensor& sparse_input)
     return compressed_tensor;
 }
 
-std::tuple<int64_t, at::Tensor> _cslt_sparse_mm_impl(
+std::tuple<at::Tensor, int64_t, int64_t, bool, int64_t> _cslt_sparse_mm_impl(
     const Tensor& compressed_A,
     const Tensor& dense_B,
     const std::optional<Tensor>& bias_opt,
     const std::optional<Tensor>& alpha_opt,
     const std::optional<c10::ScalarType> out_dtype_opt,
     bool transpose_result,
     int alg_id,
+    int split_k,
+    bool split_k_one_kernel,
```

Comment:
The search routine should return `CUSPARSELT_MATMUL_SPLIT_K_BUFFERS` (see [here](https://docs.nvidia.com/cuda/cusparselt/types.html#cusparseltmatmulalgattribute-t)) .